scripts/build_lib_ic.py

```diff
@@ -59,6 +59,10 @@ def geometry(conf):
     width += width % 200
 
     # height is the maximum required on either side
+    if len(conf['pins']) != 2:
+        raise RuntimeError("IC schematic symbols must have at least one pin "
+                           "on the left and right sides and may only have two"
+                           "sides.")
```

Comment:
Maybe just "must list pins on both sides"? or "exactly two sides" if you want to be specific.
Missing a space between "two" and "sides" here, too.